setup.py

```diff
@@ -42,15 +42,15 @@ def find_name():
 
 def find_long_description():
     """Return the content of the README.md file."""
```

Comment:
`README.md` in the docstring should be updated.

Reply:
Addressed.